pkg/modelzooserver/modelzooserver.go

```diff
@@ -15,28 +15,162 @@ package modelzooserver
 
 import (
 	"context"
+	"fmt"
+	"io"
+	"log"
 
+	"sqlflow.org/sqlflow/pkg/database"
 	pb "sqlflow.org/sqlflow/pkg/proto"
 )
 
+const modelCollTable = "sqlflow_model_zoo.model_collection"
+const modelDefTable = "sqlflow_model_zoo.model_definition"
+const trainedModelTable = "sqlflow_model_zoo.trained_model"
+
+// TODO(typhoonzero): create tables if these tables are not pre created?
+const createTableStmts = `CREATE DATABASE IF NOT EXISTS sqlflow_model_zoo;
+DROP TABLE IF EXISTS sqlflow_model_zoo.trained_model;
+DROP TABLE IF EXISTS sqlflow_model_zoo.model_definition;
+DROP TABLE IF EXISTS sqlflow_model_zoo.model_collection;
+
+CREATE TABLE sqlflow_model_zoo.model_collection (
+    id INT AUTO_INCREMENT,
+    name VARCHAR(255),
+    version VARCHAR(255),
```

Comment:
Maybe we need some field to describe the storage location?

Reply:
For creating model collection Docker images, we don't need the zip file's location because we will pack it inside the Docker image.